src/Symfony/Bundle/FrameworkBundle/Command/RouterMatchCommand.php

```diff
@@ -75,6 +76,8 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $output = new SymfonyStyle($input, $output);
```

Comment:
I'm not sure we need another name, since we're replacing the output we're using by a wrapped one, which implements the exact same interface (and an additional one, `StyleInterface`).
This output behaves exactly like the previous one.